fe/fe-core/src/main/java/com/starrocks/sql/ast/InsertStmt.java

```diff
@@ -347,43 +345,45 @@ public Table makeTableFunctionTable(SessionVariable sessionVariable) {
 
         if (format == null) {
             throw new SemanticException("format is a mandatory property. " +
-                    "Use \"format\" = \"parquet\" as only parquet format is supported now");
+                    "Use any of (parquet, orc, csv)");
+        }
+
+        if (!format.equalsIgnoreCase("parquet") && !format.equalsIgnoreCase("orc") &&
```

Comment:
It is better to maintain a map that contains all supported formats.

Reply:
done. 

I do think `target_max_file_size` is more accessible. 🤔

Reply:
done